cobalt/dom/window_on_screen_keyboard.idl

```diff
@@ -16,5 +16,5 @@
 // screen keyboard.
 
 partial interface Window {
-[ Conditional=COBALT_ENABLE_ON_SCREEN_KEYBOARD ] readonly attribute OnScreenKeyboard onScreenKeyboard;
+readonly attribute OnScreenKeyboard onScreenKeyboard;
```

Comment:
Why not just remove this web interface directly? It was conditional before anyway so Kabuki has checks to prevent JavaScript crash.